tox.ini

```diff
@@ -963,3 +963,13 @@ deps =
   pre-commit
 commands =
   pre-commit run --color=always --all-files {posargs}
+
+[testenv:typecheck]
```

Comment:
I would say we can use the name of the tool here, like mypy or pyright. For example: [.github/workflows/lint_0.yml](https://github.com/open-telemetry/opentelemetry-python/blob/main/.github/workflows/misc_0.yml#L55)

Reply:
But... The `lint` jobs only run `pylint`, and the `test` jobs run `pytest`. 😅